README.md

```diff
@@ -1,4 +1,63 @@
-# Getting Started with Create React App
+# Pomodoro Project
+
+<img src="https://user-images.githubusercontent.com/69774407/119630519-b11e2480-be17-11eb-8f89-f1179db1948e.png" width="100" height="100">
+
+#### This technique helps in managing working time as it divides it into specific periods, each period is called pomodoro, where the length of the pomodoro is 25 minutes, and after every 25 minutes there is a 5-minute break, and if four pomodoro cycles are completed, there will be a 15-minute break.
+
+## Features of this site:
+- It contains a 25-minute timer (pomodoro period).
+- Contains a pause button to timer.
+- When the timer ends, five minutes are displayed with the start button.(short break).
+- When the short break finishes (5 min) Will come back to (pomodoro period).
+## How to run it?
+#### requires [Node.js](https://nodejs.org/en/download/) Latest LTS Version: 14.17.0 to run.
+```
+cd git clone https://github.com/the-sol/pomodoro.git
+```
+```
+npm install
+```
+```
+npm start
+```
+
+## Folder structure
+
+```
+├── LICENSE
+├── README.md
+├── package-lock.json
+├── package.json
+├── public
+│   ├── favicon.ico
+│   ├── index.html
+│   ├── logo192.png
+│   ├── logo512.png
+│   ├── manifest.json
+│   └── robots.txt
+└── src
+    ├── App.css
+    ├── App.js
+    ├── App.test.js
+    ├── component
+    │   ├── StartButton
+    │   │   ├── ButtonLgic.js
+    │   │   ├── index.jsx
+    │   │   └── style.css
+    │   └── Timer
+    │       ├── TimerLogic.js
+    │       ├── index.jsx
+    │       └── style.css
+    ├── index.css
+    ├── index.js
+    ├── logo.svg
+    ├── reportWebVitals.js
+    └── setupTests.js
+```
+
+### Read [Contribution guide](CONTRIBUTING.md)
+###### SOURCE: [briandk](https://gist.github.com/briandk)
```

Comment:
Instead of writing it like this, the standard is to do something similar to the following:

```suggestion
## Contributions

We welcome and appreciate contributions of all kind to this repository. Please read the [contribution guide](CONTRIBUTING.md) for more details.
```

Check out How [React Bootstrap is writing a similar statement for example](https://github.com/react-bootstrap/react-bootstrap#contributions)

As I mentioned in another comment above, move the source to the `CONTRIBUTING.md` file.